Amazon.QLDB.Driver/transaction/AsyncTransaction.cs

```diff
@@ -126,12 +126,13 @@ internal async ValueTask DisposeAsync()
         /// </summary>
         ///
         /// <param name="statement">The PartiQL statement to be executed against QLDB.</param>
+        /// <param name="cancellationToken"> A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         ///
         /// <returns>Result from executed statement.</returns>
         ///
         /// <exception cref="AmazonServiceException">Thrown when there is an error executing against QLDB.</exception>
         /// <exception cref="QldbDriverException">Thrown when this transaction has been disposed.</exception>
-        internal async Task<IAsyncResult> Execute(string statement)
+        public async Task<IAsyncResult> Execute(string statement, CancellationToken cancellationToken = default)
```

Comment:
We have cancellationToken as a member variable for AsyncTransaction and we use that instead of passing in cancellationToken. Is this parameter needed for some reason?

Reply:
That is the API in `IAsyncExecutable`